### PR TITLE
Added About page and logout feature

### PR DESCRIPTION
**Pull request by srishti77**

### CODE REVIEW

**Review by vidhithakrar — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_about_us.xml [hunk 1]
```diff
@@ -1,57 +1,413 @@
...
+            android:textSize="@dimen/text_size_small"
+            android:textStyle="bold" />
+
+        <TableLayout
+            android:id="@+id/donation_info_international"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:stretchColumns="0,1">
+
+            <TableRow
```
Comment (vidhithakrar) [inline app/src/main/res/layout/activity_about_us.xml:188]: This TableRow looks same, can we create another layout file and include it here and then set the text?

**Review by vidhithakrar — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/pragati/karuna/home/ui/MainActivity.kt [hunk 2]
```diff
@@ -55,4 +59,23 @@ class MainActivity : FragmentResultActivity() {
...
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when(item.itemId) {
+            R.id.action_about_us -> {
+                Intent(this, AboutUsActivity::class.java).also {
+                  startActivity(it)
+                }
+            }
+            R.id.action_logout -> {
+                Intent(this, LoginActivity::class.java)
+                    .addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
```
[the remark this answers is not in the document]
Comment (srishti77) [reply app/src/main/java/com/pragati/karuna/home/ui/MainActivity.kt:80]: Done

File: app/src/main/java/com/pragati/karuna/logout/repository/LogoutRepository.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.pragati.karuna.logout.repository
+
+import com.google.firebase.auth.FirebaseAuth
+
+class LogoutRepository(private val auth: FirebaseAuth) {
```
Comment (vidhithakrar) [inline app/src/main/java/com/pragati/karuna/logout/repository/LogoutRepository.kt:5]: I think we should LoginRepository for login and logout both?

File: app/src/main/java/com/pragati/karuna/logout/repository/LogoutRepository.kt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+    fun signOut() {
```
Comment (vidhithakrar) [inline app/src/main/java/com/pragati/karuna/logout/repository/LogoutRepository.kt:7]: function name can be logout.

**Review by vidhithakrar — CHANGES_REQUESTED** [review]

**Pull request merged**